analyzer/src/funTest/kotlin/GoDepTest.kt

```diff
@@ -66,7 +66,7 @@ class GoDepTest : WordSpec() {
 
             "invoke the dependency solver if no lockfile is present and allowDynamicVersions is set" {
                 val manifestFile = File(projectsDir, "synthetic/godep/no-lockfile/Gopkg.toml")
-                val config = AnalyzerConfiguration(false, true)
+                val config = AnalyzerConfiguration(ignoreToolVersions = false, allowDynamicVersions = true)
```

Comment:
Commit message nit: It's not really ambiguous, but just not obvious.

Reply:
I was inspired by the message of the IntelliJ code inspection: "This inspection reports boolean literal arguments that could be named to remove ambiguity." I guess what they mean by ambiguity is that if a function has multiple boolean arguements, it's hard to remember the order and easy to mess it up.

Reply:
I'd argue IntelliJ's inspection wording is wrong here, because if arguments were ambiguous, the compiler would not be able to build the code. Anyway, I'll stop nit-picking here as it's then anyway a good reference to IntelliJ's inspection 😉